README.md

```diff
@@ -1,12 +1,11 @@
 # WebView
 [![WebViewControl-Avalonia](https://img.shields.io/nuget/v/WebViewControl-Avalonia.svg?style=flat&label=WebView-Avalonia)](https://www.nuget.org/packages/WebViewControl-Avalonia/)
-[![WebViewControl-WPF](https://img.shields.io/nuget/v/WebViewControl-WPF.svg?style=flat&label=WebView-WPF)](https://www.nuget.org/packages/WebViewControl-WPF/)
```

Comment:
leeav this one in the Readme ... and add a deprecated... not maintained anymore note

Reply:
added a note. Please, have a look.

Reply:
remove this line... since you added the notice below

Reply:
done